src/style.css

```diff
@@ -0,0 +1,36 @@
+
+body {
+  margin: 0;
+}
+
+.section {
+  display: flex;
+  flex-direction: column;
+  justify-content: start;
+}
+
+.stars {
+  display: flex;
+  justify-content: start;
```

Comment:
Removing this line in your demo in dev tools doesn't change anything for me, I think it's redundant.